test/unit/query-generator/add-column-query.test.ts

```diff
@@ -0,0 +1,35 @@
+import { DataTypes } from '@sequelize/core';
+import { buildInvalidOptionReceivedError } from '@sequelize/core/_non-semver-use-at-your-own-risk_/utils/check.js';
+import { expectsql, getTestDialect, sequelize } from '../../support';
+
+const dialectName = getTestDialect();
+
+describe('QueryGenerator#addColumnQuery', () => {
+  const queryGenerator = sequelize.getQueryInterface().queryGenerator;
+
+  const User = sequelize.define('User', {
+    firstName: DataTypes.STRING,
+  }, { timestamps: false });
+
+  it('generates a ADD COLUMN query in supported dialects', () => {
+    const sql = queryGenerator.addColumnQuery(User.tableName, 'age', {
+      type: DataTypes.INTEGER,
+    });
+
+    expectsql(() => sql, {
+      default: `ALTER TABLE [Users] ADD [age] INTEGER;`,
+      mssql: `ALTER TABLE [Users] ADD [age] INTEGER NULL;`,
+      postgres: `ALTER TABLE "public"."Users" ADD COLUMN "age" INTEGER;`,
+      mysql: 'ALTER TABLE `Users` ADD `age` INTEGER;',
```

Comment:
This should be the same as the default since the ticks are added automatically

```suggestion
```

Reply:
Thanks a lot, yeah I actually didn't realize this. Will keep in mind from next time!